.github/workflows/centos7-system.yml

```diff
@@ -83,65 +121,102 @@ jobs:
       with: 
         path: 'antares-version.json'
         prop_path: 'antares_xpansion_version'
-    
+
     - name: Init submodule
       run: |
            git submodule update --init --recursive antares-deps
-          
+
     - name: Install dependencies
       run: |
           pip3 install -r requirements-tests.txt
           pip3 install -r requirements-ui.txt
 
+    - name: Download userguide
+      uses: actions/download-artifact@v2
+      with:
+        name: user-guide
+        path: docs/
+
     - name: Configure
       run: |
            source /opt/rh/devtoolset-7/enable
            export LD_LIBRARY_PATH=/usr/lib64/openmpi/lib:$LD_LIBRARY_PATH
            export PATH=/usr/lib64/openmpi/bin:$PATH
-           cmake3 -B _build -S . -DDEPS_INSTALL_DIR=/rte-antares-deps-Release -DBUILD_TESTING=ON -DCMAKE_BUILD_TYPE=Release -DUSE_MPI=ON  -DCMAKE_INSTALL_PREFIX=_install -DBUILD_UI=ON
+           cmake3 -B _build -S . -DDEPS_INSTALL_DIR=/rte-antares-deps-Release -DBUILD_TESTING=ON -DCMAKE_BUILD_TYPE=Release -DUSE_MPI=ON -DCMAKE_INSTALL_PREFIX=_install -DBUILD_UI=ON -DUSER_GUIDE_PATH="docs/${{ needs.userguide.outputs.pdf-name }}"
 
     - name: Build
       run: |
            source /opt/rh/devtoolset-7/enable
            export LD_LIBRARY_PATH=/usr/lib64/openmpi/lib:$LD_LIBRARY_PATH
            export PATH=/usr/lib64/openmpi/bin:$PATH
-           cmake3 --build _build --config Release -j2  --target install
-           
+           cmake3 --build _build --config Release -j2 --target install
+
     - name: Running unit tests
       run: |
            cd _build
            ctest3 -C Release --output-on-failure -L "unit|benders"
-           
+
+    - name: .tar.gz creation
+      run: |
+        cd _build
+        cpack3 -G TGZ
+        export TGZ_NAME=$(ls *.tar.gz)
+        echo "TGZ_NAME=$TGZ_NAME" >> $GITHUB_ENV
+
+    - name: Installer .rpm creation
+      run: |
+        cd _build
+        cpack3 -G RPM
+        export RPM_NAME=$(ls *.rpm)
+        echo "RPM_NAME=$RPM_NAME" >> $GITHUB_ENV
+
     - id: create-single-file
       name: Single file .tar.gz creation
       uses: ./.github/workflows/single-file-creation-tgz
       with:
         antares-xpansion-version: ${{steps.antares-xpansion-version.outputs.prop}}
-      
-    - name: Installer .rpm creation
-      run: |
-           cd _build
-           cpack3 -G RPM
-           
-    - name: Installer .tar.gz creation
-      run: |
-           cd _build
-           cpack3 -G TGZ
 
-    - name: Installer archive upload
+    - name: Umpload  .tar.gz
```

Comment:
I think there's a typo in the name, you probably meant "Upload"

Reply:
you're right. Its been corrected